package.json

```diff
@@ -1606,6 +1606,15 @@
                     "default": null,
                     "scope": "resource"
                 },
+                "mssql.resultsMissingValueColor": {
```

Comment:
Instead of adding a config option, let's just choose a good color that we know has contrast from the vscode theme.  I'd be surprised if people really wanted full customization here, and we've had users complain that there are too many settings already. 🙃

Reply:
Yeah, lets choose a color from vscode theme. Hardcoding color doesn't work across themes.